src/oracc_cmd.ts

```diff
@@ -0,0 +1,50 @@
+import * as child_process from 'child_process';
+import { homedir } from 'os';
+import * as path from 'path';
+import * as process from 'process';
+import { promisify } from 'util';
+import * as vscode from 'vscode';
+
+const execFile = promisify(child_process.execFile);
+
+// Thin wrapper around `execFile` which prepares the environment where we want to be running
+// the commands in.
+function run_cmd(file: string,
+                 args: string[] = [],
+                 options: Object = {
+                     cwd: homedir(),
+                     env: {
+                         ...process.env,
+                         ...{PATH: '/usr/local/oracc/bin:' + process.env.PATH},
+                     },
+                 }) {
+    return execFile(file, args, options);
+}
```

Comment:
What's the point of having the `options` argument in `run_cmd`, instead of passing the default value to `execFile` in the function body? Do you anticipate us having to run different commands?

Reply:
Also, it looks like we're adding the PATH key twice in the object - I'm guessing the second one (with added `oracc`) overwrites the previous value? But I may be misunderstanding the syntax!

Reply:
> What's the point of having the `options` argument in `run_cmd`, instead of passing the default value to `execFile` in the function body? Do you anticipate us having to run different commands?

I think it could be useful for testing, if we wanted to have a local mock `oracc` command.  Not used for anything else, though.

> Also, it looks like we're adding the PATH key twice in the object - I'm guessing the second one (with added `oracc`) overwrites the previous value? But I may be misunderstanding the syntax!

Yes, the second one is overriding the default value, that was the point of these lines:
```node
> {...process.env, ...{'PATH': '/usr/local/oracc/bin'}}['PATH']
'/usr/local/oracc/bin'
```
Unless there is a cleaner way to do it.